Add Modrinth App (fixes #148) (#151)
QuiltMC/quiltmc.org: src/pages/en/install/modrinth.mdx

---
title: "Install: Modrinth"
permalink: /install/modrinth/

layout: /src/layouts/InstallPage.astro
platformKey: platform-modrinth
website: https://modrinth.com/app
---

import Message from "@atoms/Message.astro";

> The Modrinth App is a unique, open source launcher that allows you to play your favorite mods, and keep them up to date, all in one neat little package.

Reviewer: ```suggestion
The Modrinth App is a unique, open source launcher that allows you to play your favorite mods, and keep them up to date, all in one neat little package. To set up your own Quilt profile, simply follow these steps:
```
This is more consistent with the other guides.

1. Download and install the Modrinth App from [their webiste](https://modrinth.com/app)

> 2. Open Modrinth and click on the `+` in the bottom left

Reviewer: ```suggestion
2. Open the app and click on the `+` button in the bottom left
```
Saying "Open Modrinth" could be confused with opening the website.

> 3. Fill out the Name

Reviewer: ```suggestion
3. Give the profile a Name
```

> 4. Pick Quilt from the list of modloaders.

Reviewer: ```suggestion
4. Pick Quilt from the list of modloaders
```
The full-stop here is inconsistent with the other bullet points.
